src/api/controller/api/precomputed.js

```diff
@@ -65,13 +65,8 @@ export const putPrecomputed = async (ctx, id) => {
 
 export const deletePrecomputed = async (ctx, id) => {
     try {
-        const activeJob = await getActiveJob(ctx.tenant);
-        if (
-            activeJob?.data?.jobType === PRECOMPUTER &&
-            activeJob?.data?.id === id
-        ) {
-            cancelJob(ctx, PRECOMPUTER);
-        }
+        const precomputed = await ctx.precomputed.findOneById(id);
+        await cancelJob(ctx, PRECOMPUTER, precomputed?.name);
```

Comment:
Calling `cancelJob` is also useful when there is a completed job for this precomputed task, not only when there is an active job. So we should call it all the time and let the `cancelJob` function decide what needs to be done.